var/spack/repos/builtin/packages/vasp/package.py

```diff
@@ -52,6 +52,13 @@ class Vasp(MakefilePackage, CudaPackage):
         when="+vaspsol",
     )
 
+    resource(
+        name="vtsttools",
+        url="https://theory.cm.utexas.edu/code/vtstcode-209.tgz",
+        sha256="8f88265ab200ba61a3cbae119d05677e2744b5338fb9073ce6d901f38c17774b",
+        when="+vtsttools"
```

Comment:
Is there anything that maps vtstcode 209 to specific versions of vasp? Could add a range of versions to `when` here if so.

Reply:
Vtstcode is a monolithic repository that contains several versions of the vtsttools source code, each one compatible with a specific version of VASP. Each version also requires patching the VASP source code in a slightly different way. I could add the earlier VASP versions here, but would need to create a patch for each one (plus copy the correct subdirectory of vtstcode). . . .
  I will update the when to reflect that this variant isn't available for all versions.

Reply:
```suggestion
        when="@6.5.1 +vtsttools"
```
Yeah, I didn't mean to suggest creating patches for each version. Just adding the versions that have patches here is fine.

Reply:
Sounds good. VASP 6.5.1 is the current latest version anyway, so hopefully it will meet the needs of most users.